WISE-Community/WISE-Client
# chore(VLE): Convert VLE Controller to Angular

## Changes

- Converted VLEController to an Angular component
- Created Dialog components
  - Dialog With Close
  - Dialog With Confirm
  - Dialog Without Close

## Test

- Make sure the VLE works
 - Make sure steps load properly
 - Make sure navigating to other steps works
 - Make sure the project view works
 - Make sure the Notebook works
 - Make sure the Report works
- Make sure the popups work
  - Pause Screen
  - CRater "We are scoring your work"
  - Constraint when trying to go to a step that is locked
 - Make sure the VLE looks like it did before

Closes #559
